pom.xml

```diff
@@ -690,6 +690,17 @@ Contributors:
                 <artifactId>minimal-json</artifactId>
                 <version>0.9.5</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-databind</artifactId>
+                <version>2.9.9</version>
```

Comment:
Why not use latest stable release? (2.11.2)

Reply:
The reason was that I didn't change the version in this PR because, I would not impact cbor encoder/decoder in this PR.
But I'm not against changing this.

Reply:
Ok. There are probably also other dependencies that should be upgraded that could be done in one go then.

Reply:
This is updated in master now (commit 9ae537a4df120be160faaf0f66f858c5fe6e6e75)